docker/images/n8n-custom/Dockerfile

```diff
@@ -22,7 +22,8 @@ RUN rm -rf patches .npmrc *.yaml node_modules/.cache packages/**/node_modules/.c
 
 # 2. Start with a new clean image with just the code that is needed to run n8n
 FROM n8nio/base:${NODE_VERSION}
-COPY --from=builder /home/node ./
+COPY --from=builder /home/node /usr/local/lib/node_modules/n8n
+RUN ln -s /usr/local/lib/node_modules/n8n/packages/cli/bin/n8n /usr/local/bin/n8n
```

Comment:
lgtm, provided /usr/local/bin/ is in the $PATH

Reply:
looks like it is 
![image](https://user-images.githubusercontent.com/196144/208950786-44eabfa3-9859-4922-83aa-51759208e1f2.png)
![image](https://user-images.githubusercontent.com/196144/208950888-b9ed2d83-e80d-4673-809e-9bf7ebd0df22.png)